app/allocation/views/view.njk

```diff
@@ -0,0 +1,97 @@
+{% extends "layouts/two-column.njk" %}
+
+{% set additionalContainerClass = "sticky-sidebar-container" %}
+
+{% block pageTitle %}
+  {{ t("allocations::view.page_title") }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: t("actions::back_to_dashboard"),
+    classes: "app-print--hide",
+    href: dashboardUrl
+  }) }}
+
+  <a href="javascript:window.print()" class="app-!-float-right app-print--hide govuk-!-margin-top-3 app-icon app-icon--print">
+    {{ t("actions::print_move_detail") }}
+  </a>
+
+  {{ super() }}
+{% endblock %}
+
+{% block contentHeader %}
+  <header class="govuk-!-margin-bottom-8 govuk-!-padding-left-0 govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-1">
+      {{ t("allocations::view.heading", {
+        person_or_people: "person" | pluralize(allocation.resultCount),
+        moves_count: allocation.moves_count,
+        from_location: allocation.from_location.title,
+        to_location: allocation.to_location.title,
+        date: allocation.date | formatDate
+      }) | safe }}
+    </h1>
+  </header>
+{% endblock %}
+
+{% block contentMain %}
+  <section>
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.criteria_for_allocation") }}</h2>
+    <span class="govuk-caption-m govuk-!-margin-bottom-4">
+      {{ t("allocations::view.created_at") }}
+      {{ allocation.created_at | formatDateAsRelativeDay }}
+      {{ allocation.created_at | formatTime }}
+    </span>
+
+    {{ govukSummaryList(allocationDetails) }}
+
+    {% if allocation.complete_in_full and allocationPeople.emptySlots %}
+      {{ govukWarningText({
+        text: t("allocations::view.complete_in_full"),
+        iconFallbackText: "Warning"
+      }) }}
+    {% endif %}
+  </section>
+
+  <section class="govuk-!-margin-top-8">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.people.heading") }}</h2>
+    <h3 class="govuk-heading-s">{{ t("allocations::view.people.slots_filled", {
+      slots_filled: allocationPeople.filledSlots.length,
+      total_slots: allocation.moves_count,
+      person_or_people: "person" | pluralize(allocation.moves_count)
+    }) }}</h3>
+  </section>
+
+  {% for i in range(0, allocationPeople.emptySlots) -%}
+    <div class="app-border-top-1">
+      {{ appMessage({
+        classes: "app-message--muted govuk-!-margin-top-3 govuk-!-margin-bottom-3 govuk-!-padding-right-8",
+        allowDismiss: false,
+        content: {
+          html: t("allocations::view.people.empty_slot" | safe)
+        }
+      }) }}
+    </div>
+  {%- endfor %}
+
+  {% for slot in allocationPeople.filledSlots %}
+    {{ appCard(slot) }}
+  {% endfor %}
+
+  <p class="govuk-!-margin-top-9 govuk-!-margin-bottom-0">
```

Comment:
Might be worth leaving this out until it's actually working and hooked up to a journey. Otherwise it's just a `404` at this stage?

Reply:
Following up on this too - can we please remove this and add this as part of the work to introduce cancellation an allocation. 

Otherwise this just introduces a `404`.

There will be further work around support if a person has the permission to cancel too so it would be cleaner to group that work together.